src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs

```diff
@@ -43,7 +44,15 @@ protected override void OnAttached()
         {
             var rect = Control.Geometry;
 
-            _surfaceLayout = new ElmSharp.Layout(Container);
+            if (Container != null)
```

Comment:
Do you know which case Container could be null?

Reply:
```
void OnRegisterEffect(PlatformEffect effect)
{
    effect.SetContainer(Element.Parent == null ? null : Platform.GetRenderer(Element.Parent)?.NativeView);
    effect.SetControl(NativeView);
}
```

https://github.com/xamarin/Xamarin.Forms/blob/112a203f77de4807bf40eb7882b01b77af5fe51f/Xamarin.Forms.Platform.Tizen/Renderers/VisualElementRenderer.cs#L804

In this issue,  `Element.Parent` is `ShellSection`, however there is no renderer registered for `ShellSection` in `Tizen.CircularUI`. So `Container` seems to be set was set to null.

Reply:
no need to use Container, Always use XForms.NativeParent

Reply:
Fixed.